package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "unifiedpush-node-sender",
-  "version": "0.15.0",
+  "version": "0.16.0",
```

Comment:
Ok, i can update the PR to remove this version bump. When is the next release do you know? I will need to reference the vulnerability-free version of this module in another RHMAP component.